### PR TITLE
(#1062) IoCheckedBiProc.java:38-42: Avoid usage of null value in...

### PR DESCRIPTION
**Pull request by Umbrah**
This is for #1062 
I replaced usage of IoCheckedBiFunc with CheckedBiProc and added missing tests

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1107](https://github.com/yegor256/cactoos/issues/1107) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1107](https://github.com/yegor256/cactoos/issues/1107) is assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C63314D6Z/cca95b06-cbef-437a-94a3-3efacdc946af); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @llorllale[/z](https://www.0crat.com/u/llorllale) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/test/java/org/cactoos/func/IoCheckedBiProcTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ * @checkstyle JavadocMethodCheck (500 lines)
+ */
+public final class IoCheckedBiProcTest {
+    @Test(expected = IOException.class)
+    public void wrapsExceptions() throws IOException {
+        new IoCheckedBiProc<>(
+            (first, second) -> {
+                throw new Exception();
+            }
+        ).exec(true, true);
```
Comment (victornoel) [inline src/test/java/org/cactoos/func/IoCheckedBiProcTest.java]: @Umbrah please use `Assertion` and `Throws` to test for exceptions being thrown

File: src/test/java/org/cactoos/func/IoCheckedBiProcTest.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                "Must re-throw IOException",
+                () -> ex,
+                new IsEqual<>(exception)
+            ).affirm();
+        }
+    }
+
+    @Test(expected = IllegalStateException.class)
+    public void runtimeExceptionGoesOut() throws IOException {
+        new IoCheckedBiProc<>(
```
Comment (victornoel) [inline src/test/java/org/cactoos/func/IoCheckedBiProcTest.java]: @Umbrah please use `Assertion` and `Throws` to test for exceptions being thrown

File: src/test/java/org/cactoos/func/IoCheckedBiProcTest.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+import org.hamcrest.core.IsEqual;
+import org.junit.Test;
+import org.llorllale.cactoos.matchers.Assertion;
+
+/**
+ * Test case for {@link IoCheckedBiFunc}.
+ * @since 1.0
+ * @checkstyle JavadocMethodCheck (500 lines)
+ */
+public final class IoCheckedBiProcTest {
```
Comment (victornoel) [inline src/test/java/org/cactoos/func/IoCheckedBiProcTest.java]: @Umbrah also add a test for the nominal behaviour without exception being thrown

**Comment by Umbrah** [comment]
@victornoel Please check my update

**Comment by victornoel** [comment]
@llorllale it's good to merge

**Comment by llorllale** [comment]
@rultor merge

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @victornoel[/z](https://www.0crat.com/u/victornoel), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1107](https://github.com/yegor256/cactoos/issues/1107) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by sereshqua** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +20 point(s) just awarded to @victornoel[/z](https://www.0crat.com/u/victornoel)

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @sereshqua[/z](https://www.0crat.com/u/sereshqua)

**Pull request merged**